src/documentation/runners/jstorm.md

```diff
@@ -0,0 +1,102 @@
+
+## JStorm Runner prerequisites and setup
+
+The JStorm runner currently supports JStorm 2.5.0-SNAPSHOT version.
+
+You can add a dependency on the latest version of the JStorm runner by adding to your pom.xml the following:
+```java
+<dependency>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>beam-runners-jstorm</artifactId>
+  <version>{{ site.release_latest }}</version>
+</dependency>
+```
+
+### Deploying JStorm with your application
+
+To run against a Standalone cluster, you can package your program with all beam dependencies into a fat jar, and then submit the topology by following command.
+```
+jstorm jar WordCount.jar org.apache.beam.examples.WordCount --runner=org.apache.beam.runners.jstorm.JStormRunner
+```
+
+For the case that if you don't want to package a fat jar, you can upload the beam dependencies onto all cluster nodes(`$JSTORM_HOME/lib/ext/beam`) first.
+When you submit topology with argument `"--external-libs beam"`, JStorm will load the beam dependencies automatically.
+```
+jstorm jar WordCount.jar org.apache.beam.examples.WordCount --external-libs beam  --runner=org.apache.beam.runners.jstorm.JStormRunner
+```
+
+For the details of deploying JStorm cluster, please refer to [JStorm cluster deploy](http://jstorm.io/QuickStart/Deploy/index.html)
+
+## Pipeline options for the JStorm Runner
+
+When executing your pipeline with the JStorm Runner, you should consider the following pipeline options.
+
+<table class="table table-bordered">
+<tr>
+  <th>Field</th>
+  <th>Description</th>
+  <th>Default Value</th>
+</tr>
+<tr>
+  <td><code>runner</code></td>
+  <td>The pipeline runner to use. This option allows you to determine the pipeline runner at runtime.</td>
+  <td>Set to <code>JStormRunner</code> to run using JStorm.</td>
+</tr>
+<tr>
+  <td><code>topologyConfig</code></td>
+  <td>System topology config of JStorm</td>
+  <td>DefaultMapValueFactory.class</td>
+</tr>
+<tr>
+  <td><code>workerNumber</code></td>
+  <td>Worker number of topology</td>
+  <td>1</td>
+</tr>
+<tr>
+  <td><code>parallelism</code></td>
+  <td>Global parallelism number of a component</td>
+  <td>1</td>
+</tr>
+<tr>
+  <td><code>parallelismMap</code></td>
+  <td>Parallelism number of a specified composite PTransform</td>
+  <td>DefaultMapValueFactory.class</td>
+</tr>
+<tr>
+  <td><code>exactlyOnceTopology</code></td>
+  <td>Indicate if it is an exactly once topology</td>
+  <td>false</td>
+</tr>
+<tr>
+  <td><code>localMode</code></td>
+  <td>Indicate if the topology is running on local machine or distributed cluster</td>
+  <td>false</td>
+</tr>
+<tr>
+  <td><code>localModeExecuteTimeSec</code></td>
+  <td>Executing time(sec) of topology on local mode.</td>
+  <td>60</td>
+</tr>
+</table>
+
+## Additional notes
+
+### Monitoring your job
+You can monitor your job on JStorm UI while all JStorm system metrics and beam metrics can be displayed on JStorm UI.
```

Comment:
Is this still accurate?
You can monitor your job with the JStorm UI, which displays all JStorm system metrics and Beam metrics.

Reply:
Done.